source/release-notes/4.4.txt

```diff
@@ -942,6 +942,24 @@ TLS 1.3 Support
 On CentOS 8 and RHEL 8, MongoDB 4.4 (as well as versions 4.2, 4.0, and
 3.6) support TLS1.3.
 
+User to DN Mapping Exits on Network or Authentication Failures
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+:binary:`~bin.mongod`, :binary:`~bin.mongos`, or
+:binary:`~bin.mongoldap` returns an error if one of the user to
+Distinguished Name (DN) mappings defined with the following
+configuration settings or command line options cannot be evaluated due
+to networking or authentication failures to the LDAP server:
+
+- :setting:`security.ldap.userToDNMapping`
+- :option:`mongod --ldapUserToDNMapping`
+- :option:`mongos --ldapUserToDNMapping`
+- :option:`mongoldap --ldapUserToDNMapping`
+
+:binary:`~bin.mongod`, :binary:`~bin.mongos`, or
+:binary:`~bin.mongoldap` rejects the connection request and does not
+check the remaining documents in the array.
```

Comment:
Small nit -- in this context, our users wouldn't really know what array we're referring to here.  

I'd probably go with "rejects the connection request and does not check the remaining mappings, if any. " or something like that.

We might also consider breaking up the first sentence.  

:binary:`...` returns an error if ... mapings cannot be evaluated due to networking or auth failures to the LDAP server.  The ... rejects the connection request and does not...

To specify user to DN maping, see:
<list>

Reply:
@kay-kim updated